sdk/storage/platform-matrix.json

```diff
@@ -7,32 +7,32 @@
   ],
   "include": [
     {
-      "Agent": { "macos-11": { "OSVmImage": "macos-11", "Pool": "Azure Pipelines" } },
+      "Agent": { "macos-11": { "OSVmImage": "env:MACVMIMAGE", "Pool": "env:MACPOOL" } },
```

Comment:
Storage has their own pools due to the requirement for them being in a different region, not sure this is handling that.

Reply:
Good catch, I need to update the regex